lib/DismissiblePrompt/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { node, string, func } from 'prop-types';
+import Icon from '../Icon';
+import Button from '../Button';
+
+const DismissiblePrompt = ({ children, onDismiss, className, ...rest }) => (
+  <div {...rest} className={`dismissible-prompt ${className}`}>
```

Comment:
Gonna get accessible-cray here. Could we add these roles + attributes to this div. It just makes screen readers understand it a bit better.

There are bigger accessibility issues are modals/dialogs, but this at least makes things better going forward!

<img width="811" alt="Screenshot 2020-03-10 at 14 49 33" src="https://user-images.githubusercontent.com/33638789/76324776-641b6480-62de-11ea-9c07-39fee5875dc2.png">


(I stole it from here: https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html)

Reply:
Just for visibility, cleared things up on slack. this guy isnt a modal, just a little box that lives along side content rather than overlaying anything. We spread the rest of the props so if you wanted to use it that way you can always add these in!